CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # @primer/components
 
+## 29.1.1
+
+### Patch Changes
+
+- [#1441](https://github.com/primer/react/pull/1441) [`f3f5afb9`](https://github.com/primer/react/commit/f3f5afb9cf91e0ce3ea287351c871eb1559f1fa4) Thanks [@jfuchs](https://github.com/jfuchs)! - Exclude storybook stories from TypeScript compilation for builds
```

Comment:
Let's update reword this to highlight the impact of this change (i.e. it fixes the required `css` prop error 🎉 )